### PR TITLE
Fix/5 - Possible cheat on digit answers

### LINKED ISSUE #5: Possible cheat on digit answers
# Type of issue

- [ ] Feature
- [x] Fix
- [ ] Other

# Description

If a user adds a letter to an answer containing only digits, he can succeed in forcing the calculation of the Sørensen-Dice coefficient. Depending on the tolerated typo rate used (default or not), the answer can finally be considered valid when it should not. Example:

```
import answerIsValid from 'quiz-answer-validator';

const response = answerIsValid('10.000.000.00a', ['10.000.000.000']);

console.log(response); // returns true
```

### PR DESCRIPTION
**Pull request by ehenon**
# Description

Fixes #5

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My branch is up-to-date with the target branch

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -67,9 +67,9 @@ export default function answerIsValid(
     return purifiedValidAnswers.includes(purifiedInputAnswer);
   }
 
-  const similarityCoefficients: number[] = purifiedValidAnswers.map(
-    (answer) => getSørensenDiceCoefficientBetween(answer, purifiedInputAnswer),
-  );
+  const similarityCoefficients: number[] = purifiedValidAnswers
+    .filter((answer) => /^\d+$/.test(answer) === false)
```
Comment (soranoc) [inline src/index.ts:71]: Peut être sortir ce test de regex dans une méthode puisque déjà utilisé en ligne 64

Comment (ehenon) [reply src/index.ts:71]: Tu veux faire une méthode `containsOnlyDigits` ?

Comment (soranoc) [reply src/index.ts:71]: Oui par exemple

Comment (ehenon) [reply src/index.ts]: ✔️

**Review by soranoc — APPROVED** [review]

**Pull request merged**